### PR TITLE
Incorporate Blue feedback

### PR DESCRIPTION
**Pull request by karcass**
### Explain pull request

Please provide a clear and concise reason for this pull request and the impact of the change

### Is this a breaking change

A breaking change would require consumers or implementors of the API to modify their code for it to continue to function (ex: renaming of a required field or the change in data type of an existing field). A non-breaking change would allow existing code to continue to function (ex: addition of an optional field or the creation of a new optional endpoint). 

* [ ] Yes, breaking
* [ ] No, not breaking
* [ ] I'm not sure

### `Provider` or `agency`

Which API(s) will this pull request impact:

* [ ] `provider`
* [ ] `agency`
* [ ] both

### CODE REVIEW

File: policy/README.md [hunk 1]
```diff
@@ -65,67 +65,69 @@ Policies should be re-fetched whenever (a) a Policy expires, or (b) at an interv
...
 
-Rules are ordered most-general to most-specific.  E.g. a “later” cap would take precedence over an “earlier” cap.  The internal mechanics of ordering are up to the Policy editing and hosting software.
+Rules are ordered most-specific to most-general.  E.g. an “earlier” cap rule would take precedence over a “later” cap rule.  The internal mechanics of ordering are up to the Policy editing and hosting software.
 
 <a name="messages"></a>
 ### Messages
 
 Some Policies as established by the Agency may benefit from rider communication.  This optional field contains a map of languages to be (optionally?) shown to the user.  (Unanswered question: shown with what frequency?  At sign-up?  Once per day?)
 
+Langugate tag values will be per [BCP 47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt).
```
Comment (avatarneil) [inline policy/README.md]: S.P. `Langugate` -> `Language`

File: policy/README.md [hunk 2]
```diff
@@ -65,67 +65,69 @@ Policies should be re-fetched whenever (a) a Policy expires, or (b) at an interv
...
-| `start_time` | time | O | Beginning time-of-day (hh:mm:ss) when the rule is in effect (default 00:00:00) |
-| `end_time` | time | O | Ending time-of-day (hh:mm:ss) when the rule is in effect (default 23:59:59) |
-| `days` | day[] | O | Days `["sun", "mon", "tue", "wed", "thu", "fri", "sat"]` when the rule is in effect (default all) |
-| `messages` | { string:string } | O | Message to rider user, if desired, in various languages, keyed by I18n (see [Messages](#messages))|
-| `value_url ` | URL | O | URL to an API endpoint that can provide dynamic information for the measured value (see [Value URL](#value-url)) |
+| Name         | Type     | R/O | Description      |
+| ---          | ---      | --- | ---              |
+| `name`       | String   | R   | Name of cap item |
+| `rule_type`  | enum     | R   | Type of policy (see [“policy types”](#policy-types))|
+| `rule_units` | enum     | O   | Measured units of policy (see [“policy units”](#policy-units))|
```
Comment (avatarneil) [inline policy/README.md]: Shouldn't be optional if we have no default units, however there are some cases where we don't care (like count); I'll try and brainstorm some better ways we could define this. Also the descriptions on this and the previous line should be updated to link to #rule-types and #rule-units

File: policy/README.md (hunk 1 above)
Comment (karcass) [reply policy/README.md]: fixed

File: policy/README.md (hunk 2 above)
Comment (karcass) [reply policy/README.md]: fixed types/units

Comment (karcass) [reply policy/README.md]: units of `count` could be "vehicles"

units of `user` could be "n/a"

**Pull request merged**